packages/grid/test/column-group.test.js

```diff
@@ -79,76 +79,6 @@ describe('column group', () => {
     expect(columns[1].frozen).to.be.true;
   });
 
-  it('should hide group column', () => {
-    columns[0].hidden = true;
-    columns[1].hidden = true;
-
-    expect(group.hidden).to.be.true;
-  });
-
-  it('should unhide group column', () => {
```

Comment:
Removed this test, as the assumption is wrong. If the group has been set to hidden explicitely, then it should **not** unhide when a column is made visible. That should only happen if the group is auto-hidden.

Added respective test cases below.

Reply:
Hmm...if you set a group `hidden`, it automatically hides its children also, which makes sense: a hidden group should be fully invisible, similarly to if you set a `<div>` hidden, its children will not get rendered.

Now, if you unhide one column in the group, you end up in an (illegal) state where the group is hidden, but one of its children is visible.

Maybe we should refactor the logic so that a change in a group's `hidden` property doesn't affect the `hidden` property of its children, but effectively, when a group is marked `hidden`, none of the child columns get rendered?